src/components/Toolbar/Toolbar.js

```diff
@@ -247,10 +247,9 @@ export default class Toolbar extends Component {
 
         <Transition className={namespace} component="span">
           {isPanelActive && this.state.showContent ? (
-            <aside
-              className={`${namespace}__panel`}
+            <nav
```

Comment:
Related to [IBM 2.4.1 Bypass Blocks](http://www.ibm.com/able/guidelines/ci162/bypass_blocks.html)

> Multiple elements with "nav" role do not have unique labels

Here's some notes from that checkpoint in the a11y checker plugin:

> What to do
> For each element with a "nav" role, either set the value of the aria-labelledby attribute to the id value of any visible text that appropriately and uniquely describes that navigation region;
> OR, use the aria-label attribute to add an invisible, unique, non-empty label to an element with the "nav" role.

In this case, it may be helpful to call out that this particular `nav` is related to whatever toolbar section ("Settings", etc.) opened it.

Also I think it would be nice to go ahead and add a label to the "primary" `nav` as well, containing those primary icon buttons (again, "Settings" etc).

Reply:
Sounds good, and I like the second suggestion! Feel free to take over if it's something you're interested in and have the bandwidth, otherwise, I'll likely be able to revisit later on in the week